ksqldb-rest-model/src/test/java/io/confluent/ksql/rest/entity/SourceDescriptionTest.java

```diff
@@ -53,117 +54,117 @@ public void shouldImplementHashCodeAndEqualsProperty() {
                     SOME_STRING, Optional.empty(), readQueries, writeQueries, fields, SOME_STRING,
                     SOME_STRING, SOME_STRING, SOME_STRING,
                     SOME_BOOL, SOME_STRING, SOME_STRING, SOME_STRING, SOME_INT, SOME_INT,
-                    SOME_STRING),
+                    SOME_STRING, Optional.empty()),
```

Comment:
we should add at least one test where this isn't empty to make sure equals/hashcode properly include it